src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -11,15 +11,15 @@ ferc_to_sqlite_settings:
   ferc2_dbf_to_sqlite_settings:
     years: [2019, 2020]
   ferc2_xbrl_to_sqlite_settings:
-    years: [2021, 2022]
+    years: [2021, 2023]
```

Comment:
Keep these years synchronized with the FERC 1 & FERC 714 so that when we're testing them we actually know if the most recent year works.